system_tests/tests/test_full_system.py

```diff
@@ -16,20 +45,36 @@ def truncate_db():
     yield
 
 
-def test_create_get():
-    response = requests.post(
-        WRITE_URL,
-        json={
+@pytest.fixture()
+def data():
+    yield copy.deepcopy(
+        {
             "user_id": 1,
             "information": {},
             "locked_fields": {},
             "events": [{"type": "create", "fqid": "a/1", "fields": {"f": 1}}],
-        },
+        }
     )
-    assert response.status_code == 201
+
+
+def test_create_get(data):
+    response = requests.post(WRITE_URL, json=data,)
+    assert_response_code(response, 201)
+
     response = requests.post(GET_URL, json={"fqid": "a/1"})
     assert response.json() == {
         "f": 1,
         "meta_deleted": False,
         "meta_position": 1,
     }
+
+
+def test_create_delete_get(data):
+    data["events"].append({"type": "delete", "fqid": "a/1"})
+    response = requests.post(WRITE_URL, json=data,)
+    assert_response_code(response, 201)
+
+    response = requests.post(GET_URL, json={"fqid": "a/1"})
+    assert_error_response(
+        FlaskResponseWrapper(response), ERROR_CODES.MODEL_DOES_NOT_EXIST
```

Comment:
DO you use the `FlaskResponseWrapper` only to use `assert_error_response`? Maybe you should create a new set of `assert_*` for fsts or check the type in `assert_error_response` nad behave differently.